fcrepo-indexing-solr/pom.xml

```diff
@@ -33,6 +33,11 @@
       <groupId>org.apache.camel</groupId>
       <artifactId>camel-http4</artifactId>
     </dependency>
+
+    <dependency>
+      <groupId>org.apache.camel</groupId>
```

Comment:
I am not seeing where `camel-restlet` is used in `fcrepo-indexing-solr`.
I removed this dependency locally, and the build completes successfully.
If there is no particular reason for this dependency here, let's drop it.